src/css/components/navigation.less

```diff
@@ -26,6 +26,7 @@
 .navbar-inverse .navbar-nav > .active > a:focus {
   color: @black-haze;
   background-color: transparent;
+  cursor: pointer;
```

Comment:
You're right, but I need to investigate why we need an explicit override here – I thought it came from BS.